sdk/src/ssi/did.go

```diff
@@ -4,12 +4,13 @@ import (
 	"github.com/TBD54566975/ssi-sdk/crypto"
 	"github.com/TBD54566975/ssi-sdk/did"
 	"github.com/goccy/go-json"
+	"github.com/lestrrat-go/jwx/jwk"
 	"github.com/pkg/errors"
 )
 
-type DIDKeyWrapper struct {
-	PrivateJSONWebKey []byte
-	DIDKey            string
+type generateDIDKeyResult struct {
+	DidKey string  `json:"didKey" validate:"required"`
+	Jwk    jwk.Key `json:"jwk" validate:"required"`
```

Comment:
I think it's important to signify that this represents the private key, and not the public one. 
```suggestion
	PrivateJSONWebKey jwk.Key `json:"privateJwk"`
```